### ISSUE TITLE
Update Performance Benchmarks in Docs and README

### ISSUE THREAD

**Issue by navidcy** [CliMA/Oceananigans.jl#1676]
We should run the [Performance Benchmarks](https://clima.github.io/OceananigansDocumentation/stable/benchmarks/) with the latest Oceananigans.jl version and on Julia v1.6.

**Comment by hennyg888** [comment]
@francispoulin and I are already working on that.
We should be able to have the new benchmarks out and up around some time next week.

**Comment by francispoulin** [comment]
I see that in the Benchmarks there are 5 sets of results.  Do we want to only do these or would we like to include other results?  

There are a lot of benchmark examples we could include, but I agree it's probably better to focus on the most interesting ones.

**Comment by ali-ramadhan** [comment]
Yeah I agree we should probably just include the most interesting ones for now.

Eventually I was hoping to upgrade the benchmark scripts to also produce plots (bar charts?) to make it easier to visually parse all the numbers/information, then it might make sense to include more benchmark results.

**Comment by francispoulin** [comment]
Thanks @ali-ramadhan .  I think it would be worthwhile for us to see the results from all the benchmarks so I asked @henryg888 to try running all the scripts. If we see anything interesting we could all more results as they are generated, otherwise we will update the 5.

At the moment we are waiting for julia to be updated on the servers and then we can do proper benchmarking on the V100's. 

I agree that plots would be nice and maybe we can play with that too.  Shouldn't be difficult but will certainly discuss it with you.

**Comment by francispoulin** [comment]
Resolved by #1722
